### DISCUSSION TITLE
Drawing artifacts with split creation and resizing

### DISCUSSION THREAD

**Discussion by edrozenberg** [ghostty-org/ghostty#3944] [Feature Requests, Ideas] [answered]
Ghostty Version 1.0.0
Build 8469
Commit https://github.com/ghostty-org/ghostty/commit/4b4d4062dfed7b37424c7210d1230242c709e990
(MacOS)

When creating and resizing splits, there are temporary, noticeable drawing artifacts.

https://github.com/user-attachments/assets/c82e30c8-17a5-49f8-b3bd-18ff8100663b

**Comment by mitchellh** [comment, marked as answer]
This is a dupe.
